### PR TITLE
Add Delorean CLI and skopeo to test base image

### PR DESCRIPTION
**Pull request by mikenairn**
# Description
Add Delorean CLI and skopeo to base test image
Add test-containers checks to test the format of the test-containers.yaml file (test-containers/check/format) and that all images exist (test-containers/check/images).
All test-containers checks can be executed with test-containers/check

```
make test-containers/check
```

Jiras:

* https://issues.redhat.com/browse/DEL-289
* https://issues.redhat.com/browse/DEL-287

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: make/test-containers.mk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+.PHONY: test-containers/check/format
+test-containers/check/format:
+	yq v test-containers.yaml
```
Comment (b1zzu) [inline make/test-containers.mk:4]: Should we document somewhere how to fix the format of the test-containers.yaml if this job fails?

Comment (mikenairn) [reply make/test-containers.mk:4]: :man_shrugging: It's just checking it's valid yaml tbh, if it fails make it valid yaml

Comment (b1zzu) [reply make/test-containers.mk:4]: I thought it was a linter

**Pull request merged**